sdk/tests/conformance2/extensions/webgl-shader-pixel-local-storage.html

```diff
@@ -50,6 +50,7 @@
   if (!gl) {
     return function() {
       testFailed("WebGL2 context does not exist");
+      finishTest();
     }
   }
```

Comment:
This part seems incorrect. Should probably be:
```js
if (!gl) {
  testFailed("WebGL context does not exist");
  finishTest();
  return;
}
```

Reply:
Agreed, thanks, fixed.

Reply:
An extra return from the block is needed to exit the function early.

Reply:
Thanks again, fixed.